baseband/mark4/frame.py

```diff
@@ -69,6 +69,7 @@ class Mark4Frame(VLBIFrameBase):
 
     _header_class = Mark4Header
     _payload_class = Mark4Payload
+    invalid_data_value = 0.
```

Comment:
Why is this needed? Doesn't it inherit it from `VLBIFrameBase`?
... though possibly this is a good way for it to appear in the docs...

Reply:
Ahh, yeah, I'll remove it. It's only in VDIF's `frame.py` because of `frameset`, but I got confused.

Reply:
Still to be removed?